tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{38,39,310,311,yapf,isort,flake8}
+envlist = py{38,39,310,311},yapf,isort,flake8
```

Comment:
additional fix 2:

previous envlist resulsts to `pyyapf, pyisort, pyflake8` so `[testenv:yapf]` setting is not used and run default `coverage run -m unittest discover -s test`.

```
$ poetry run tox
py38: commands[0]> coverage run -m unittest discover -s test
py38: OK ✔ in 1.65 seconds
py39: commands[0]> coverage run -m unittest discover -s test
py39: OK ✔ in 1.44 seconds
py310: commands[0]> coverage run -m unittest discover -s test
py310: OK ✔ in 1.44 seconds
py311: commands[0]> coverage run -m unittest discover -s test
py311: OK ✔ in 1.4 seconds
pyyapf: commands[0]> coverage run -m unittest discover -s test
pyyapf: OK ✔ in 1.43 seconds
pyisort: commands[0]> coverage run -m unittest discover -s test
pyisort: OK ✔ in 1.43 seconds
pyflake8: commands[0]> coverage run -m unittest discover -s test
```

now:

```
$ poetry run tox
py38: commands[0]> coverage run -m unittest discover -s test
py38: OK ✔ in 3.92 seconds
py39: commands[0]> coverage run -m unittest discover -s test
py39: OK ✔ in 1.46 seconds
py310: commands[0]> coverage run -m unittest discover -s test
py310: OK ✔ in 1.45 seconds
py311: commands[0]> coverage run -m unittest discover -s test
py311: OK ✔ in 1.44 seconds
yapf: commands[0]> yapf -dr .
yapf: OK ✔ in 1.67 seconds
isort: commands[0]> isort -c ./kannon ./test
isort: OK ✔ in 0.39 seconds
flake8: commands[0]> pflake8 ./kannon ./test
```